packages/project/src/trpc/project-router.ts

```diff
@@ -63,6 +63,38 @@ export const projectRouter = router({
 
     return projectIds.map((project) => project.id);
   }),
+
+  userPublishCount: procedure.query(async ({ ctx }) => {
+    try {
+      if (
+        ctx.authorization.type !== "user" &&
+        ctx.authorization.type !== "token"
+      ) {
+        throw new Error("Not authorized");
```

Comment:
I would do early returns everywhere here to reduce the nesting level, the giant try/catch is reducing readability in my subjective opinion

Reply:
try/catch will be there anyway to catch postgrest error